requirements.txt

```diff
@@ -9,4 +9,4 @@ psycopg2-binary
 -e .[compatible-mypy]
 
 # Overrides:
-mypy==0.982
+mypy==0.990
```

Comment:
Unless mypy 0.991 solves the issues we're having, IMO we can make a release that's being tested with `mypy==0.982`.

Having a release that solves *some* mypy 0.99x problems is better than delaying the release.

Of course having full support for latest mypy would be ideal, but right now I don't have time and interest to look into the CI failures.